integration/spark/src/test/scala/org/apache/spark/util/CarbonCommandSuite.scala

```diff
@@ -82,6 +83,43 @@ class CarbonCommandSuite extends QueryTest with BeforeAndAfterAll {
        """.stripMargin)
   }
 
+  protected def createTestTable(tableName: String): Unit = {
+    sql(
+      s"""
```

Comment:
Instead of adding new testcase ,
In one of the existing testcases of **loading, insert into, partition table loading, partition table insert into**, just add a validation for segment id

Reply:
Can you help point out which test case I can add validation? Because there are too many.

Reply:
can you add in TestLoadGeneral/TestLoadDataWithAutoLoadMerge/InsertIntoCarbonTableTestCase. For partition - StandardPartitionTableLoadingTestCase